spec/core/v2/ics-002-client-semantics/README.md

```diff
@@ -343,44 +343,44 @@ to intervene to unfreeze a frozen client & provide a new correct ClientMessage w
 It is utilised to verify presence or absence of a particular key/value pair in state
 at a particular finalised height (necessarily associated with a particular commitment root).
 
+#### `Counterparty`
+
+`Counterparty` is the data structure responsible for maintaining the counterparty information.
+
+```typescript
+interface Counterparty {
+    channelId: Identifier
```

Comment:
Shall we change all the channelID references to to clientIDs?

Reply:
I think we should keep this in ICS4 actually, since it is only used for channel level packet processing. In IBC-go we have the channel ID and clientID separate to enable aliasing.

So in the spec I think we should have them separate to note it isn't a strict requirement to have the identifiers the same. In a note, we can say that implementations (like Solidity) can choose to keep them the same.

Reply:
Makes definitely sense to me!